proto/frequenz/api/common/v1/metrics/metrics.proto

```diff
@@ -186,7 +186,7 @@ enum Metric {
 //     command, clients need to request current values within the bounds.
 message MetricSample {
   // The UTC timestamp of when the metric was sampled.
-  google.protobuf.Timestamp sampled_at = 1;
+  google.protobuf.Timestamp origin_time = 1;
 
   // The metric that was sampled.
```

Comment:
collected?

Reply:
As discussed, I updated it to `sampled_time`.